hack/test-end-to-end.sh

```diff
@@ -134,49 +170,35 @@ echo "[INFO] Pushed centos7"
 
 # Process template and apply
 echo "[INFO] Submitting application template json for processing..."
-osc process -n test -f examples/sample-app/application-template-${BUILD_TYPE}build.json > "${CONFIG_FILE}"
+osc process -n test -f examples/sample-app/application-template-stibuild.json > "${STI_CONFIG_FILE}"
```

Comment:
I left that one because we use it in the readme and i didn't want to confuse people by using an "sti" namespace in the sample, people might assume it means more than it does.